ui/pages/settings/settings-search-list/settings-search-list.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
```

Comment:
It would be best if the newly added components were functional components instead of class components. We can either fix that in this PR, or create a ticket to refactr that code, that is picked up immediately after this PR is merged

The relevant components are this one and `settings-search.js`

Reply:
Done